src/components/00-materials/README.md

```diff
@@ -88,13 +88,30 @@ npm install @svgr/webpack --save-dev
 
 ## Contribution
 
-### Process of adding a new icon/image
+### Process of adding a new image
 
 1. Involve a designer by sending him the new SVG in question, unless the designer already has the file. He will review it and possibly change some things, for example by adjusting inner padding etc. in order to guarantee overall quality.
-2. Remove unnecessary code inside the SVG:
+1. Remove unnecessary code inside the SVG:
```

Comment:
`1.` is a line above. This should be `2.`

Reply:
All the numbers are wrong in this document.

Reply:
https://guides.github.com/features/mastering-markdown/

See "ordered lists".